### PR TITLE
Document how to suggest new plugins

### PR DESCRIPTION
**Pull request by stefanalund**
#### :heavy_check_mark: Checklist

- [x] Relevant documentation updated
- [x] Prettier run on changed files

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/plugin_template.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+name: 'Plugin suggestion'
+about: 'Suggest a new Backstage plugin'
+title: '[Plugin] THE PLUGIN NAME'
+labels: plugin
+---
+
+<!--- Thanks for suggesting a new plugin! -->
+<!--- If you plan to work on this plugin, please Assign yourself or your team member(s). If not, please add the 'help wanted' label -->
```
Comment (Rugvip) [inline .github/ISSUE_TEMPLATE/plugin_template.md:9]: Most ppl reporting issues are not allowed to add labels, maybe ask to mention in the issue instead?

File: .github/ISSUE_TEMPLATE/bug_template.md [hunk 2]
```diff
@@ -1,35 +1,43 @@
 ---
 name: 'Bug Report'
 about: 'Create Bug Report'
+labels: bug
```
Comment (Rugvip) [inline .github/ISSUE_TEMPLATE/bug_template.md:4]: 👍

**Review by Rugvip — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/plugin_template.md (hunk 1 above)
Comment (stefanalund) [reply .github/ISSUE_TEMPLATE/plugin_template.md:9]: Good point @Rugvip -- do you think they are allowed to assign themselves?

Comment (OrkoHunter) [reply .github/ISSUE_TEMPLATE/plugin_template.md]: @stefanalund I can not assign myself to any issue. Only the `maintainers` of the repository can assign

**Pull request merged**